static/tests/backend/specs/api/comments.js

```diff
@@ -199,7 +200,8 @@ describe('create comment API broadcast', function(){
 
   // NOTE: this hook will timeout if you don't run your Etherpad in
   // loadTest mode. Be sure to adjust your settings.json when running
-  // this test suite
+  // this test suite. Also, set ALLOW_LOAD_TESTING=true in the .env file
```

Comment:
`run your Etherpad in loadTest mode.` é a mesma coisa de `ALLOW_LOAD_TESTING=true`...Com o `also, set` tá parecendo que são duas coisas diferentes